player/public/js/ekstepRendererApi.js

```diff
@@ -7,6 +7,9 @@
  * @author Vinu Kumar <vinu.kumat@tarento.com>
  */
 window.EkstepRendererAPI = {
+	isMobile: function(){
+		return (/version\//i.test(navigator.userAgent.toLowerCase()))
```

Comment:
Chech for both android & version to indentify is it mobileView. Presently you are relaying on only version.